src/scss/06_components/pages/_news-page.scss

```diff
@@ -1,6 +1,6 @@
 // Fix sidebar width when there are only short news titles
 // TODO: REGIONS Clean this up with other region things
-$-card-teaser-img-aspect-ratio: 3 / 2;
+$-card-teaser-img-aspect-ratio: 3 * 0.5;
```

Comment:
This does not make sense, I think we should use the Math.div here instead. 3 * 0.5 is not a nice aspect ration :)

Reply:
Yeah, let's fix this 👍